docs/cli/cli.mdx

```diff
@@ -24,61 +24,61 @@ xata [command] [subcommand] [options]
-- [project create](/reference/cli/project#create) - Create a new project
-- [project delete](/reference/cli/project#delete) - Delete a project
-- [project init](/reference/cli/project#init) - Link a project to the folder
-- [project get](/reference/cli/project#get) - Get a field from a project description
+- [init](/cli/init#init) - Link a project to the current folder
+- [project list](/cli/project#list) - List all projects
+- [project describe](/cli/project#describe) - Describe a project
+- [project create](/cli/project#create) - Create a new project
+- [project delete](/cli/project#delete) - Delete a project
+- [project init](/cli/project#init) - Link a project to the folder
+- [project get](/cli/project#get) - Get a field from a project description
 
 ### Organization Management
-- [organization list](/reference/cli/organization#list) - List all organizations
-- [organization describe](/reference/cli/organization#describe) - Describe an organization
-- [organization create](/reference/cli/organization#create) - Create a new organization
-- [organization delete](/reference/cli/organization#delete) - Delete an organization
-- [organization get](/reference/cli/organization#get) - Get a field from an organization description
+- [organization list](/cli/organization#list) - List all organizations
+- [organization describe](/cli/organization#describe) - Describe an organization
+- [organization create](/cli/organization#create) - Create a new organization
+- [organization delete](/cli/organization#delete) - Delete an organization
+- [organization get](/cli/organization#get) - Get a field from an organization description
 
 ### Branch Management
-- [branch list](/reference/cli/branch#list) - List all branches
-- [branch describe](/reference/cli/branch#describe) - Describe a branch
-- [branch create](/reference/cli/branch#create) - Create a new branch
-- [branch delete](/reference/cli/branch#delete) - Delete a branch
-- [branch url](/reference/cli/branch#url) - Print URL (connection string) for a branch
-- [branch checkout](/reference/cli/branch#checkout) - Checkout a branch
-- [branch tree](/reference/cli/branch#tree) - List all branches as a tree
-- [branch get](/reference/cli/branch#get) - Get a field from a branch description
-- [branch wait-ready](/reference/cli/branch#wait-ready) - Wait for a branch to be ready
+- [branch list](/cli/branch#list) - List all branches
+- [branch describe](/cli/branch#describe) - Describe a branch
+- [branch create](/cli/branch#create) - Create a new branch
+- [branch delete](/cli/branch#delete) - Delete a branch
+- [branch url](/cli/branch#url) - Print URL (connection string) for a branch
+- [branch checkout](/cli/branch#checkout) - Checkout a branch
+- [branch tree](/cli/branch#tree) - List all branches as a tree
+- [branch get](/cli/branch#get) - Get a field from a branch description
+- [branch wait-ready](/cli/branch#wait-ready) - Wait for a branch to be ready
 
 ### Schema Migrations
```

Comment:
Nit: order of the list of these comments should be start/ migrate before complete etc OR lexicographically sorted. They are random right now, feels off 🤔